packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -97,6 +97,12 @@ The client-based rate limit framework can operate in one of three modes:
 | **Log per client**                          | The rate limit is based on the [org-wide rate limit](/docs/reference/rate-limits/) values, but the client-specific rate limit violation information is logged as System Log events. |
 | **Do nothing**                                | Rate limits aren't enforced at the client-specific level. The rate limit is based on the [org-wide rate limit](/docs/reference/rate-limits/) values. No new or additional System Log events are produced from this feature in this mode. |
 
+#### Per client rate limit
+
+As of October 12, 2021, the default client-based rate limit for the OAuth 2.0 `/authorize` and `/login/login.htm` endpoints is set to **Log per client** mode. This means that if an org's client-based rate limit was previously set to **Do nothing**, the setting is changed to **Log per client** mode.
```

Comment:
Lets add a statement that also says if you are already in "enforce and log mode" then it will remain untouched.